src-ts/lib/styles/_typography.scss

```diff
@@ -1,3 +1,4 @@
+@import "./breakpoints";
```

Comment:
when importing stylesheets, the "./" isn't necessary. so this can just be `@import "breakpoints"`